.bcr/config.yml

```diff
@@ -0,0 +1,3 @@
+fixedReleaser:
+  login: luispadron
+  email: heyluispadron@gmail.com
```

Comment:
For now we'll need to mark a single person as releaser. See an example PR for what this does:

- PR Example: https://github.com/bazelbuild/bazel-central-registry/pull/1010
- Information on this config: https://github.com/bazel-contrib/publish-to-bcr#a-note-on-release-automation

In the future we could create a bot account to handle this.

Reply:
Perhaps for rules_ios the release author is a shared google group email?

Reply:
Makes sense, though it needs to be a GitHub account with a fork of the BCR since that's how it makes the PR to release. If you want to make that account that would be great and I can get this PR updated